### DISCUSSION TITLE
What should I do

### DISCUSSION THREAD

**Discussion by Ankushbadyal1** [rapid7/metasploit-framework#17359] [Q&A]
![Screenshot_2022-12-12-17-37-12-399_com.termux.jpg](https://user-images.githubusercontent.com/100316052/207041373-7a906853-9845-4fc0-b9ee-99f818eb596f.jpg)

**Comment by bcoles** [comment]
You could try `bundle install`.

Note that Termux is not a supported platform for Metasploit.

**Reply by Ankushbadyal1** [reply]
![Screenshot_2022-12-12-18-01-31-321_com.termux.jpg](https://user-images.githubusercontent.com/100316052/207045878-f9f85dd6-7e10-49ad-9379-c1a00f333ae9.jpg)
